runtime/runtime-params-estimator/src/gas_cost.rs

```diff
@@ -30,24 +35,40 @@ pub(crate) struct GasCost {
 }
 
 pub(crate) struct GasClock {
-    start: Consumed,
+    start: Option<Instant>,
```

Comment:
```suggestion
    start: Instant,
```

I think it's fine to just always track the time.